src/build.py

```diff
@@ -705,9 +724,26 @@ def CopyLLVMTools(build_dir, prefix=''):
       CopyLibraryToArchive(os.path.join(build_dir, 'lib', e), prefix)
 
 
+def WinBuildEnv(build_dir, use_gnuwin32=False, bin_subdir=False,
+                runtime='Release'):
+  if not IsWindows():
+    return None
+  cc_env = host_toolchains.SetUpVSEnv(build_dir)
+  if use_gnuwin32:
+    cc_env['PATH'] = cc_env['PATH'] + os.pathsep + os.path.join(GNUWIN32_DIR, 
```

Comment:
Is there a subtle difference between `a + os.pathsep + os.path.join(b, c)` and `os.path.join(a, b, c)` ?